terraform/environments/digital-prison-reporting/main.tf

```diff
@@ -226,6 +226,8 @@ module "glue_hive_table_creation_job" {
     "--extra-jars"                = local.glue_jobs_latest_jar_location
     "--class"                     = "uk.gov.justice.digital.job.HiveTableCreationJob"
     "--dpr.aws.region"            = local.account_region
+    "--dpr.config.s3.bucket"      = module.s3_glue_job_bucket.bucket_id,
+#    "--dpr.config.key"            = "(Required) config key. Will be specified at point of call in the reload step-functions"
```

Comment:
It says Required but is commented out?

Reply:
Left the comment there for documentation purpose. The `dpr.config.key` arg is left out of the job definition to avoid having to default to a random domain but it needs to be supplied at the point of execution.